src/contrib/serializers/Akka.Serialization.Hyperion.Tests/HyperionConfigTests.cs

```diff
@@ -160,6 +162,33 @@ public void Hyperion_serializer_should_read_cross_platform_package_name_override
 #endif
             }
         }
+        
+        [Fact]
+        public void Hyperion_serializer_should_allow_to_setup_surrogates()
+        {
+            var config = ConfigurationFactory.ParseString(@"
+                akka.actor {
+                    serializers.hyperion = ""Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion""
+                    serialization-bindings {
+                        ""System.Object"" = hyperion
+                    }
+                    serialization-settings.hyperion {
+                        surrogates = [
+                            ""Akka.Serialization.Hyperion.Tests.FooHyperionSurrogate, Akka.Serialization.Hyperion.Tests""
+                        ]
+                    }
+                }
+            ");
+            using (var system = ActorSystem.Create(nameof(HyperionConfigTests), config))
+            {
+                var serializer = (HyperionSerializer)system.Serialization.FindSerializerForType(typeof(object));
+                var expected = new Foo("bar");
+                var serialized = serializer.ToBinary(expected);
+                var deserialized = serializer.FromBinary<Foo>(serialized);
+                deserialized.Bar.Should().Be("bar.");
```

Comment:
Need to validate that we actually use the surrogate here, somewhere

Reply:
Done.